### PR TITLE
Setup CI for rnaseq_cgl pipeline (resolves #214)

### LINKED ISSUE #214: Setup CI for rna-seq CGL

### PR DESCRIPTION
**Pull request by jvivian**

### CODE REVIEW

**Comment by jvivian** [comment]
@hannes-ucsc Ready.  I have a follow-up ticket to fix dynamic disk allocation, and create another to fix some refactoring that I want to do.

File: src/toil_scripts/rnaseq_cgl/test/test_rnaseq_cgl.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    finally:
+        shutil.rmtree(subdir)
+
+
+def create_config(path):
+    """Creates config file for test at path"""
+    fpath = os.path.join(path, 'config.txt')
+    with open(fpath, 'w') as f:
+        f.write('ci_test,'
+                'https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/rnaseq_cgl/ci/test_sample_chr6.tar.gz')
```
Comment (hannes-ucsc) [inline src/toil_scripts/rnaseq_cgl/test/test_rnaseq_cgl.py]: NL missing at EOL

**Comment by hannes-ucsc** [comment]
@jvivian, Can you fix the missing NL at EOF in your next PR?

**Comment by jvivian** [comment]
>  Can you fix the missing NL at EOF in your next PR? 

Definitely

**Pull request merged**